CarpetLib/src/restrict_3d_rf2.cc

```diff
@@ -110,8 +110,8 @@ void restrict_3d_rf2(T const *restrict const src,
 
   } else {
 
-// Loop over coarse region
-#pragma omp parallel
+    // Loop over coarse region
+    // #pragma omp parallel
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.